lib/plugins/bed.js

```diff
@@ -10,35 +10,64 @@ function noop (err) {
 
 function inject (bot) {
   bot.isSleeping = false
+  const mcData = require('minecraft-data')(bot.version)
+
   const beds = new Set(['white_bed', 'orange_bed', 'magenta_bed', 'light_blue_bed', 'yellow_bed', 'lime_bed', 'pink_bed', 'gray_bed',
```

Comment:
Isn't it the same id just with another damage value?

Reply:
No it's not